presto-docs/src/main/sphinx/release/release-0.100.rst

```diff
@@ -22,3 +22,5 @@ General Changes
 * Rename config property ``task.shard.max-threads`` to ``task.max-worker-threads``.
   This property sets the number of threads used to concurrently process splits.
   The old property name is deprecated and will be removed in a future release.
+* Fix referencing ``NULL`` values in :ref:`row_type`.
+* Make :ref:`map_type` comparable.
```

Comment:
Can you also add the following: 
* Fix leak of tasks blocked during query teardown

After that I think we are good to go.